htdocs/accountancy/class/lettering.class.php

```diff
@@ -154,7 +154,7 @@ public function lettrageTiers($socid)
 					$sql .= " INNER JOIN " . MAIN_DB_PREFIX . "accounting_bookkeeping as bk ON (bk.fk_doc = pay.fk_bank AND bk.code_journal='" . $obj->code_journal . "')";
 					$sql .= " WHERE payfac.fk_paiement = '" . $obj->url_id . "' ";
 					$sql .= " AND bk.code_journal IN (SELECT code FROM " . MAIN_DB_PREFIX . "accounting_journal WHERE nature=4 AND entity=".$conf->entity.") ";
-					$sql .= " AND fac.entity = ".$conf->entity;
+					$sql .= " AND fac.entity IN (".getEntity('invoice').")";
```

Comment:
In the module accountancy, we must always use entity = ... and no IN getEntity, because in accountancy module, we must never use the sharing (otherwise, invoices will be accounted in 2 differents ledgers of 2 differents companies).
getentity must be used everywhere except pages or class of 'accountancy' module